test/bin/manage_build_cache.sh

```diff
@@ -101,18 +101,22 @@ action_upload() {
     echo "Uploading ${iso_size} of ISO images to '${iso_dest}'"
     run_aws_cli s3 sync --quiet --include '*.iso' "${iso_base}" "${iso_dest}"
 
-    # Upload ostree commits
-    local -r repo_src="${src_base}/repo.tar"
-    local -r repo_dst="${dst_base}/repo.tar"
+    # Upload brew-rpms and repo archives
+    for dir in brew-rpms repo ; do
+        local repo_src="${src_base}/${dir}.tar"
+        local repo_dst="${dst_base}/${dir}.tar"
```

Comment:
Shall we change the prefixes from `repo_`? Was confused in later lines why we're removing repo :P